plantcv/plantcv/print_results.py

```diff
@@ -1,18 +1,169 @@
 # Print Numerical Data
```

Comment:
For `print_results` I think the output order of the data is not guaranteed to be in the same order as the header because dictionaries are unordered. We could ensure the order by using the header list but I think we can get away with the natural dictionary order. For shape output for example:

```python
if 'shapes' in outputs.measurements:
    header = ["HEADER_SHAPES"]
    data = ["SHAPES_DATA"]
    for k, v in outputs.measurements['shapes'].items():
        header.append(k)
        data.append(v)
    result.write('\t'.join(map(str, header)) + '\n')
    result.write('\t'.join(map(str, data)) + '\n')
```